docs/source/changelog.md

```diff
@@ -6,7 +6,60 @@ command line for details.
 
 ## [Unreleased]
 
-## [2.0.0]
+## 2.0
+
+### [2.0.1]
+
+([full changelog](https://github.com/jupyterhub/jupyterhub/compare/2.0.0...2.0.1))
+
+2.0.1 is a bugfix release, with some additional small improvements,
+especially in the new RBAC handling and admin page.
+
+Several issues are fixed where users might not have the
+default 'user' role as expected.
+
+### 2.0.0...master@{2021-12-22}
```

Comment:
Delete this line and this LGTM for merge and release!

Reply:
Ooops, copypasta. Thanks for your vigilance!